.github/workflows/test.yaml

```diff
@@ -101,9 +101,21 @@ jobs:
           with:
             cache: ${{ github.event.inputs.cache == 'true' || github.event.inputs.cache == '' }}
 
+  typecheck:
```

Comment:
Can you align the name with what is done in Panel?

https://github.com/holoviz/panel/blob/09f7a4c11ccce79612c3d43c8a62d85615693e97/pixi.toml#L203-L204
https://github.com/holoviz/panel/blob/09f7a4c11ccce79612c3d43c8a62d85615693e97/.github/workflows/test.yaml#L255-L273

Reply:
OK